full-service/src/db/transaction_log.rs

```diff
@@ -405,13 +402,6 @@ impl TransactionLogModel for TransactionLog {
         // Verify that the account exists.
         Account::get(&AccountID(account_id_hex.to_string()), conn)?;
 
-        // Verify that the TxProposal is well-formed according to our
-        // assumptions about how to store the sent data in our wallet
-        // (num_output_TXOs = num_outlays + change_TXO).
-        if tx_proposal.tx.prefix.outputs.len() - tx_proposal.outlays.len() > 1 {
```

Comment:
What's the reason this invariant no longer holds?

Reply:
We can have multiple change txos once mixed token transactions are enabled, and I also cannot think of a scenario where this is even possible when there is a single token type in the transaction, so it seems unnecessary to check for.